pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -950,7 +952,11 @@ func describeDeployments(f formatter, dcNode *deploygraph.DeploymentConfigNode,
 	}
 
 	for i, deployment := range deploymentsToPrint {
-		out = append(out, describeDeploymentStatus(deployment.ReplicationController, i == 0, dcNode.DeploymentConfig.Spec.Test, restartCount))
+		if latestName == deployment.ReplicationController.Name {
+			out = append(out, describeDeploymentStatus(deployment.ReplicationController, i == 0, dcNode.DeploymentConfig.Spec.Test, restartCount))
+		} else {
+			out = append(out, describeDeploymentStatus(deployment.ReplicationController, i == 0, dcNode.DeploymentConfig.Spec.Test, 0))
```

Comment:
@kargakis maybe I don't understand your original question... let say you have active deployment that have pods (active means it is currently being rolled out, right?) and you want to know if it has pods are restarting... so when the deployment complete, you will see the restarts, correct? If it fails, then it means you will see failed deployment (or we revert you to previous deployment).